### PR TITLE
Org Browser + Multi Select Files

### PR DESCRIPTION
**Pull request by sbudhirajadoc**
Also https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000NGcs7YAD/view
and
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000IYEpOYAX/view

### What does this PR do?
We have updated docs to include information about --
1. Multi-selecting files and folders in explorer for Source Deploy/Retrieve
2. Fields and field details for custom objects.

### Functionality After
Relevant doc updates were made.

### CODE REVIEW

**Comment by sbudhirajadoc** [comment]
@jeffb-sfdc this is early because I am in training next week, so please do not approve until feature release. Thanks!

**Review by jeffb-sfdc — APPROVED** [review]

File: docs/_articles/en/user-guide/development-models.md [hunk 1]
```diff
@@ -95,13 +95,13 @@ When you deploy the code changes, the local version of the source files overwrit
...
 
-You can deploy source of a manifest, a source file, a directory, or a folder:
+You can deploy source of a manifest, source files, directories, or folders:
 
 - Manifest
   - In VS Code explorer or editor, right-click a manifest file and select **SFDX: Deploy Source in Manifest in Org**.
   - With a manifest file open in the editor, open the Command Palette and run **SFDX: Deploy Source in Manifest in Org**.
 - Source File or Directory
-- In VS Code explorer, right-click a source file or a directory and select **SFDX: Deploy Source to Org**.
+- In VS Code explorer, right-click single or multi-selected source files or directories and select **SFDX: Deploy Source to Org**.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/development-models.md:104]: How does the user select multiple files, is it through right-click?

Comment (sbudhirajadoc) [reply docs/_articles/en/user-guide/development-models.md:104]: the standard explorer way -- shift select.

Comment (ekapner) [inline docs/_articles/en/user-guide/development-models.md:104]:
```suggestion
- In Org Browser explorer, right-click a single file or directory, or select multiple source files or directories, then select **SFDX: Deploy Source to Org**.
```

File: docs/_articles/en/user-guide/development-models.md [hunk 2]
```diff
@@ -125,15 +125,17 @@ Org Browser displays the available metadata types and their corresponding compon
 
 ### Retrieve Source
 
-You can retrieve a component or multiple components to your local project from the default org. To do so, click the retrieve button next to the component or the metadata type.
+You can retrieve a component or multiple components to your local project from the default org:
+- In the org browser, click the retrieve button next to the component or the metadata type.
```
Comment (ekapner) [inline docs/_articles/en/user-guide/development-models.md:129]:
```suggestion
- In Org Browser, click the retrieve button next to the component or the metadata type.
```

File: docs/_articles/en/user-guide/org-browser.md [hunk 3]
```diff
@@ -25,7 +25,11 @@ When you open the Org Browser, all the metadata for the default org is saved in
 
 ## Listing and Refreshing Metadata
 
-The Org Browser lists all the metadata types **(4)** in your default org. To view the components **(5)**, expand the metadata type. When you expand metadata types with folders such as Reports, Dashboards, Documents, and EmailTemplates, all the available folders are displayed. You can view the components in a folder by expanding it.
+The Org Browser lists all the metadata types **(4)** in your default org. To view the components **(5)**, expand the metadata type. When you expand metadata types with folders such as Reports, Dashboards, Documents, and EmailTemplates, all the available folders are displayed. You can view the components in a folder by expanding it. Expand the Custom Object metadata type to view fields and their type information. Additional information is displayed for following field types:
+-  strings: string(length)
+-  textarea: textarea(length)
+-  email: email(length)
+-  Lookup: lookup(reference)
```
Comment (ekapner) [inline docs/_articles/en/user-guide/org-browser.md:32]: Is "Lookup" capitalized?

**Review by ekapner — COMMENTED** [review]
Just some minor stuff.

File: docs/_articles/en/user-guide/org-browser.md (hunk 3 above)
Comment (jeffb-sfdc) [reply docs/_articles/en/user-guide/org-browser.md:32]: "Lookup" is more of an alias, for when an object has a foreign-key relationship with another object.  For example, the `Campaign` object has `Parent (reference)` and `Owner (reference)`, so my suggestion would be to change the docs to `lookup: lookup(reference)`.  Also, we should probably pluralize each:
```
-  strings: string(length)
-  textareas: textarea(length)
-  emails: email(length)
-  lookups: lookup(reference)
```

Comment (ekapner) [reply docs/_articles/en/user-guide/org-browser.md]: Thanks @jeffb-sfdc for the explanation!

**Review by ekapner — APPROVED** [review]

**Pull request merged**